### PR TITLE
warm-up exercise

### PR DESCRIPTION
**Pull request by RemyaManoharan**

### CODE REVIEW

File: nodejs/week1/avg.js [hunk 1]
```diff
@@ -0,0 +1,14 @@
+const newArguments = process.argv.slice(2);
+if (newArguments.length === 0) {
+  console.log("Provide some nembers");
+} else {
+  // convert the arguments to numbers using function Number and filter the non numbers
+  const numbers = newArguments.map(Number).filter((num) => !isNaN(num));
```
Comment (NatsGt) [inline nodejs/week1/avg.js:6]: Nice solution to convert strings into numbers and then remove all NaN.

File: nodejs/week1/avg.js [hunk 2]
```diff
@@ -0,0 +1,14 @@
...
+  if (numbers.length === 0) {
+    console.log("Provide some valid numbers");
+  } else {
+    const sum = numbers.reduce((acc, cur) => acc + cur, 0);
+    const avg = sum / numbers.length;
+    console.log(`Average is  ${avg}`);
```
Comment (NatsGt) [inline nodejs/week1/avg.js:12]: I really like your solution for this assignment! Way to go!

**Review by NatsGt — APPROVED** [review]

File: nodejs/week1/avg.js [hunk 3]
```diff
@@ -0,0 +1,14 @@
+const newArguments = process.argv.slice(2);
+if (newArguments.length === 0) {
+  console.log("Provide some nembers");
+} else {
+  // convert the arguments to numbers using function Number and filter the non numbers
+  const numbers = newArguments.map(Number).filter((num) => !isNaN(num));
+  if (numbers.length === 0) {
+    console.log("Provide some valid numbers");
```
Comment (raihanmehran) [inline nodejs/week1/avg.js:8]: Well, good job :) 
But just filtering out the characters might not be the case. But what I would suggest that alerting the user to know that he/she has made a mistake, will be a good option.

File: nodejs/week1/avg.js (hunk 2 above)
Comment (RemyaManoharan) [reply nodejs/week1/avg.js:12]: Thanks NatsGt for the feedback

**Pull request merged**